massa-versioning/src/versioning_ser_der.rs

```diff
@@ -123,10 +109,14 @@ impl MipInfoDeserializer {
     pub fn new() -> Self {
         Self {
             u32_deserializer: U32VarIntDeserializer::new(Included(0), Excluded(u32::MAX)),
-            len_deserializer: U32VarIntDeserializer::new(
+            name_len_deserializer: U32VarIntDeserializer::new(
                 Included(0),
                 Excluded(MIP_INFO_NAME_MAX_LEN),
             ),
+            components_len_deserializer: U32VarIntDeserializer::new(
+                Included(0),
+                Excluded(MIP_INFO_COMPONENTS_MAX_ENTRIES),
```

Comment:
I think in the codebase we almost always use Included() on MAX constants, is there a reason to exclude the max value here (and above)?

Reply:
Because the constant name is max entries and we start at 0 so I would say it makes sense to use Excluded here no?